src/components/Event/Event.st.css

```diff
@@ -42,7 +47,7 @@
 }
 
 .root {
-    -st-states: selected, fullday, rtl;
+    -st-states: selected, fullday, rtl, disabled;
```

Comment:
I don't see that the `disabled` state is used anywhere.
If it's not used it can be removed